Vladyslav-Hudz/README.md

```diff
@@ -0,0 +1,44 @@
+# Arduino - Get Started Guide
+
+This document will provide you with the initial instructions for using Arduino and getting started with the platform.
+
+## Requirements
+
+Before getting started, you will need:
+
+- An Arduino board
+- A computer with Arduino IDE installed
+- A USB cable to connect Arduino to the computer
+
+## Step 1: Installing Arduino IDE
+
+1. Visit the official Arduino website at [arduino.cc](https://www.arduino.cc/).
+2. Download the Arduino IDE from the "Downloads" page.
+3. Install the Arduino IDE on your computer by following the instructions for your operating system.
+
+## Step 2: Connecting Arduino
+
+1. Connect Arduino to your computer using the USB cable.
+2. Launch the Arduino IDE.
+
+## Step 3: Selecting the Board and Port
+
+1. In the `Menu Bar` in the `Tools` menu, select the appropriate Arduino board under `Board`.
+2. Choose the correct `Port` for your Arduino board in the `Tools` menu. If you're unsure which port to select, refer to the documentation specific to your board.
+
+![The view of the Arduino IDE](https://images.theengineeringprojects.com/image/main/2018/10/Introduction-to-Arduino-IDE-9.jpg)
+
+## Step 4: First Project
+
+Now you're ready to write your first code for Arduino.
+
+1. Select `File` > `Examples` to browse through the available code examples.
```

Comment:
Describe the purpose before the action. For example: _To view available code examples, select File > Examples._